<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowTitle">
   <string>Advance Options</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="paramTab">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="tokenEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="endpointLabel">
         <property name="text">
          <string>Endpoint</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="cnrelayEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="endpointEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="tokenLabel">
         <property name="text">
          <string>Token</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="cnrelayLabel">
         <property name="text">
          <string>CNRelay</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="otherLabel">
         <property name="text">
          <string>Other</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="otherEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>-option1 parameter1
-option2 parameter2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="envTab">
      <attribute name="title">
       <string>Environment Variables</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPlainTextEdit" name="envEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>VARIABLE_NAME_1=variable_value_1
VARIABLE_NAME_2=variable_value_2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
